### PR TITLE
gnss external update

### PR DESCRIPTION
**Pull request by m-kuhn**
- @lucienicolier do you have a screenshot to add here?

### CODE REVIEW

File: en/fieldwork/gps.rst [hunk 1]
```diff
@@ -26,8 +26,17 @@ Inside the positioning menu you can turn on the positioning display which will s
...
     If you see WGS 84 lat/lon information instead of information in your project CRS, you probably have no signal yet.
 
-Using external GNSS-Reciever
-----------------------------
+Using an external GNSS-Receiver
+-------------------------------
+
+QField supports connecting external GNSS antennas via bluetooth.
+
+In settings -> positioning, connected bluetooth devices will show up and can be chosen as position source.
```
Comment (nirvn) [inline en/fieldwork/gps.rst:34]:
```suggestion
In settings -> positioning, paired bluetooth devices can be scanned and chosen as position source.
```

**Comment by lucienicolier** [comment]
@m-kuhn I should receive 2 more tablets and 2 GNSS antennas this week. I can do better screenshots. I let you know when they are here.

**Comment by lindacamathias** [comment]
```
tlmgr: Local TeX Live (2020) is older than remote repository (2021).
Cross release updates are only supported with
  update-tlmgr-latest(.sh/.exe) --update
See https://tug.org/texlive/upgrade.html for details.
Error: Process completed with exit code 1.
```
@m-kuhn, @mbernasocchi says you know what to do :)

**Comment by m-kuhn** [comment]
I fixed it a week ago, should be good after this new workflow run

**Pull request merged**